internal/app/connectconformance/results.go

```diff
@@ -547,3 +535,118 @@ func indent(s string) string {
 	}
 	return strings.Join(lines, "\n")
 }
+
+func checkWireDetails(testCaseReq *conformancev1.ClientCompatRequest, expected, actual *conformancev1.ClientResponseResult) multiErrors {
+	expectedWire := expected.WireDetails
+	actualWire := actual.WireDetails
+	if actualWire == nil {
+		return nil
+	}
+	var errs multiErrors
+
+	if expectedWire.GetActualStatusCode() != 0 {
+		if diff := cmp.Diff(expectedWire.ActualStatusCode, actualWire.ActualStatusCode, protocmp.Transform()); diff != "" {
+			errs = append(errs, fmt.Errorf("actual HTTP status code does not match: - wanted, + got\n%s", diff))
+		}
+	}
+
+	switch testCaseReq.Protocol {
+	case conformancev1.Protocol_PROTOCOL_CONNECT:
+		if len(actualWire.ActualHttpTrailers) > 0 {
```

Comment:
Maybe a dumb nit, but should we document or validate anywhere that `expected.ActualHttpTrailers` are unused. In other words, if you were to put them in a test yaml file in the `expectedResponse` nothing is actually looking at them. Just a thought, no biggie though.

Reply:
Yeah, they should not be set in an expected response. See last paragraph [here](https://github.com/connectrpc/conformance/pull/780#discussion_r1483557941). WDYT, should we change up the protos to make it more clear?

Reply:
(modulo the status code which you mentioned above is the only thing we'd keep around in the response)

Reply:
Ok. See #784